Modules/_zstd/compressor.c

```diff
@@ -699,7 +699,7 @@ static PyType_Slot zstdcompressor_slots[] = {
     {0}
 };
 
-PyType_Spec zstdcompressor_type_spec = {
+PyType_Spec zstd_compressor_type_spec = {
     .name = "_zstd.ZstdCompressor",
     .basicsize = sizeof(ZstdCompressor),
     .flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_HAVE_GC,
```

Comment:
Let's add the `Py_TPFLAGS_IMMUTABLETYPE` at the same time. This will make the type read-only.

Reply:
See https://github.com/python/cpython/pull/133775#discussion_r2082316012, intended for a follow-up

Reply:
#133784